### PR TITLE
Add routes for index and create rents.

### PR DESCRIPTION
**Pull request by fedecvz**
## Summary:
Add routes for index and create rents.

## Trello card:
https://trello.com/c/WaFgVyMn/32-crear-endpoint-de-creacion-de-rent-e-index

### CODE REVIEW

File: app/controllers/api/v1/rents_controller.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+module Api
+  module V1
+    class RentsController < ApiController
+      def create
+        rent = Rent.create(rents_create_params)
```
Comment (juanisierra) [inline app/controllers/api/v1/rents_controller.rb]: What happens if a param is missing?

**Review by juanisierra — CHANGES_REQUESTED** [review]

File: .env [hunk 2]
```diff
@@ -0,0 +1,5 @@
+MAILER_DOMAIN=smtp.mailtrap.io
```
Comment (juanisierra) [inline .env]: You should have added this before to the .gitignore so that this file hadn't been uploaded

**Review by juanisierra — CHANGES_REQUESTED** [review]

File: app/controllers/api/v1/rents_controller.rb [hunk 3]
```diff
@@ -0,0 +1,25 @@
+module Api
+  module V1
+    class RentsController < ApiController
+      def create
+        rent = Rent.new(rents_create_params)
+        rent.save!
```
Comment (juanisierra) [inline app/controllers/api/v1/rents_controller.rb]: What if it fails?

File: config/routes.rb [hunk 4]
```diff
@@ -2,6 +2,7 @@
   namespace :api do
     namespace :v1 do
       resources :books, only: [:show, :index]
+      resources :rents, only: [:create, :index], path: 'users/:user_id/rents'
```
Comment (juanisierra) [inline config/routes.rb]: If rents is defined inside the user resource it should already have this path

**Review by juanisierra — CHANGES_REQUESTED** [review]

File: .env (hunk 2 above)
Comment (fedecvz) [reply .env]: Done!

Comment (fedecvz) [reply .env]: I deleted .env

**Review by juanisierra — APPROVED** [review]

**Pull request closed without merging**